types/index.d.ts

```diff
@@ -1,8 +1,13 @@
 import { ExecException, ChildProcess } from 'child_process'
+import fastFolderSizeSync from './sync'
 
 declare function fastFolderSize(
   path: string,
   callback: (err: ExecException | null, bytes?: number) => void
 ): ChildProcess
 
 export = fastFolderSize
+
+declare module 'fast-folder-size/sync' {
+  export = fastFolderSizeSync
+}
```

Comment:
despite the tests passing, I have a feeling that defining the types this way will not let a typescript consumer use it in the way you intend it:

```suggestion
import fastFolderSizeSync from 'fast-folder-size/sync'
// or
const fastFolderSizeSync = require('fast-folder-size/sync')
```

Reply:
I think it's correct, what are you proposing?

Reply:
I'm happy to merge and release a new version. Will you have a chance to try the released package to see if the typings work correctly?

Reply:
Okay I will!